### ISSUE TITLE
App name: "Thorium" to "Thorium Browser"?

### ISSUE THREAD

**Issue by ylluminate** [Alex313031/thorium#732]
I've noticed that Thorium Browser is conflicting with the Thorium eBook reader. Is there any chance we could have the asset that is compiled and installed via Homebrew named to "Thorium Browser.app" instead of just "Thorium.app?"

**Comment by dm5n** [comment]
Yes, just downloaded this Browser and on attemtped install on mac silicon macos 15.5 - getting this same conflict. Thorium browser FTW please
